docs/installers.rst

```diff
@@ -627,16 +630,26 @@ Currently, the following tasks are implemented:
 *   wine: ``wineexec`` Runs a windows executable. Parameters are
     ``executable`` (``file ID`` or path), ``args`` (optional arguments passed
     to the executable), ``prefix`` (optional WINEPREFIX),
-    ``arch`` (optional WINEARCH, required when you created win64 prefix), ``blocking`` (if true, do not run the process in a thread), ``working_dir`` (optional working directory), ``include_processes``  (optional space-separated list of processes to include to
-    being watched)
-    ``exclude_processes`` (optional space-separated list of processes to exclude from
-    being watched), ``env`` (optional environment variables), ``overrides`` (optional DLL overrides).
+    ``arch`` (optional WINEARCH value, by default inherited from the `game:` section, which itself defaults to win64. The value can be set to ``win32`` to run the task in a 32-bit prefix.),
+    ``blocking`` (if true, do not run the process in a thread), 
+    ``description`` (a message be shown to the user during the execution of the task), 
+    ``working_dir`` (optional working directory), 
+    ``exclude_processes`` (optional space-separated list of processes to exclude from being monitored when determening if the execute phase finished), 
+    ``include_processes`` (the opposite of ``exclude_processes``, is used to override Lutris' built-in monitoring exclusition list), 
```

Comment:
Typo: exclusition -> exclusion

Reply:
God, I really shouldn't have merged this half awake